ui.tests/test-module/specs/customFunctionTest.spec.js

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+ * ADOBE CONFIDENTIAL
+ * ___________________
+ *
+ * Copyright 2023 Adobe Systems Incorporated
+ * All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Adobe Systems Incorporated and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Adobe Systems Incorporated and its
+ * suppliers and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Adobe Systems Incorporated.
+ */
+
+describe('Form with custom functions configured in client lib', () => {
+    const pagePath = "/content/forms/af/core-components-it/samples/af2-custom-function/basic.html";
+    let formContainer = null;
+
+    /**
+     * initialization of form container before every test
+     * */
+    beforeEach(() => {
+        cy.previewForm(pagePath).then(p => {
+            formContainer = p;
+        })
+    });
+
+    if (cy.af.isLatestAddon()) {
```

Comment:
Since it is part of release, this check needs to be removed now. Can you remove it and re-create the PR ?

Reply:
done.